### PR TITLE
[FX][EZ] Fix tuple type annotations

### PR DESCRIPTION
**Pull request by jamesr66a**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #51816 [WIP][FX] Normalize torch.nn.functional calls and standard Module calls
* **#52010 [FX][EZ] Fix tuple type annotations**
* #51998 [FX] Fix leaf modules in Transformer

I typed the wrong thing here and accidentally made the type annotation expect a 1-element tuple

Differential Revision: [D26355481](https://our.internmc.facebook.com/intern/diff/D26355481)

### CODE REVIEW

**Review by ansley — APPROVED** [review]

**Review by Chillee — APPROVED** [review]
LGTM
